segments/admin.py

```diff
@@ -11,13 +11,10 @@ class SegmentAdmin(admin.ModelAdmin):
     """
 
     prepopulated_fields = {"slug": ("name",)}
-    list_display = ('name', 'members_count', 'definition', 'manager_method')
-    readonly_fields = ('created_date', 'members_count', 'static_users_sample', 'sql_users_sample')
-    fields = ('name', 'slug', 'members_count', 'definition', 'manager_name', 'manager_method',
-              'sql_users_sample', 'static_ids', 'static_users_sample', 'created_date')
-
-    def members_count(self, segment):
-        return len(segment)
+    list_display = ('name', 'priority', 'members_count', 'definition')
+    readonly_fields = ('created_date', 'members_count', 'updated_date', 'recalculated_date')
+    fields = ('name', 'slug', 'priority', 'members_count', 'definition', 'created_date', 'updated_date', 'recalculated_date')
```

Comment:
Why do we need the `fields` here ? Django will add it itself

Reply:
I'm using the fields to define the order.